playbooks/roles/fix_broken/tasks/ubuntu.yml

```
@@ -4,7 +4,7 @@
   become: true
   shell: ps aux | grep "apt update" | grep -v grep | wc -l
   register: result
-  retries: 30
+  retries: 10
   delay: 10
   until: result.stdout | int == 0
```

I'm not sure I understand, but this doesn't look like the right approach. These loops should exit when the `until` condition is met, right? Reducing the retry limit would just make the playbook fail faster, not complete successfully sooner(?)